terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -5,23 +5,26 @@ log_level = "debug"
 chs_url = "https://cidev.aws.chdev.org"
 piwik_url = "https://matomo.platform.aws.chdev.org/"
 piwik_site_id = "7"
+piwik_start_goal_id = "20"
+piwik_update_start_goal_id = "23"
 cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
 cookie_domain = "cidev.aws.chdev.org"
 account_local_url = "https://account.cidev.aws.chdev.org"
-piwik_start_goal_id = "20"
```

Comment:
Just rearranged where these values appear. Makes things a bit clearer and more closely matches how they were laid out in chs-configs.